building-with-ocean/deploying-components/deploying-aquarius.md

```diff
@@ -12,11 +12,7 @@ Aquarius consists of two parts:\
 
 * A server for hosting Aquarius. See [this guide](setup-server.md) on creating a server.
 * Docker and Docker compose are installed. Click [here](https://docs.docker.com/engine/install/) to view guide on installing docker.
-* Ethereum API. Aquarius uses Ethereum api for monitoring on-chain events.\
-  Choose any api provider of your choice. Some of the commonly used are:
-  * [Infura](https://infura.io/)
-  * [Alchemy](https://www.alchemy.com/)
-  * [Moralis](https://moralis.io/)
+* [Obtain an API key](../using-ocean-libraries/configuration.md#obtaining-api-key-for-ethereum-node-provider)
```

Comment:
Why have you added a link to the page that was removed?

Reply:
Seems to be in multiple places

Reply:
this link exists: (https://ocean-protocol.gitbook.io/using-ocean-lib/building-with-ocean/using-ocean-libraries/configuration#obtaining-api-key-for-ethereum-node-provider).